### PR TITLE
Progress Report 32

### PR DESCRIPTION
**Pull request by ike709**
Needs to go through review

### CODE REVIEW

**Comment by mirrorcult** [comment]
i would put the new maps first tbh, i think we did that for packed as well

File: content/post/22-03-27-progress-report-32.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
++++
+title = "Progress Report #32: Geoff"
+author = "DrSmugleaf, ike709, & Timrod"
+date = 2022-03-23
```
Comment (ike709) [inline content/post/22-03-27-progress-report-32.md:4]:
```suggestion
date = 2022-03-27
```

File: content/post/22-03-27-progress-report-32.md [hunk 2]
```diff
@@ -0,0 +1,250 @@
++++
+title = "Progress Report #32: Geoff"
+author = "DrSmugleaf, ike709, & Timrod"
```
Comment (PaulRitter) [inline content/post/22-03-27-progress-report-32.md:3]:
```suggestion
author = "DrSmugleaf, ike709 & Timrod"
```

File: content/post/22-03-27-progress-report-32.md [hunk 3]
```diff
@@ -0,0 +1,250 @@
...
+categories = [
+    "Progress Report"
+]
+reddit = ""
+thumbnail = "pr_32"
++++
+As promised in the last progress report, here's the second half of our two-part special! This part covers February 2022.
+<!--more-->
+
+TODO Thumbnail
```
Comment (PaulRitter) [inline content/post/22-03-27-progress-report-32.md:14]:
```suggestion
```

**Review by PaulRitter — APPROVED** [review]
looks very good, good job

File: content/post/22-03-27-progress-report-32.md (hunk 2 above)
Comment (PaulRitter) [reply content/post/22-03-27-progress-report-32.md:3]: i would understand , and, but , & just looks like ass

Comment (ike709) [reply content/post/22-03-27-progress-report-32.md:3]: https://owl.purdue.edu/owl/research_and_citation/apa_style/apa_formatting_and_style_guide/reference_list_basic_rules.html

Comment (PaulRitter) [reply content/post/22-03-27-progress-report-32.md:3]: the apa can say whatever they want, its not gonna make it look less like someone collapsed on their keyboard

**Pull request merged**